.github/workflows/tests.yml

```diff
@@ -83,6 +90,11 @@ jobs:
       with:
         name: ${{ matrix.browser }}-mac-testrun.log
         path: testrun.log
+    - uses: actions/upload-artifact@v1
```

Comment:
Why not put everything into `test-results` folder? E.g. `test-results/results.json`, `test-results/testrun.log`.

Reply:
Good call.